README.md

```diff
@@ -205,6 +208,55 @@ gh api \
 jq '[.[] | {login}]'
 ```
 
+This generates the file from all users in a GitHub organization. The same
+concept could also be used for a GitHub team. Of course, the JSON file can be
+generated by any means, as long as it follow the expected format.
+
+##### Inputs
+
+One of `team`, `local-members-path`, or `members-url` must be set. If multiple
+of these inputs are set, only one will be used. Which one will be used is
+undefined, but deterministic.
+
+**team**: This is actually the org and team formateted as ${org}/${team-slug}
```

Comment:
Can we specify multiple teams? In the repositories we (Cloud SDK) maintain, it's norm to have the service team and the Cloud SDK team maintain a repository together (e.g., [java-storage](https://github.com/googleapis/java-storage))

Reply:
@suztomo We really just need one team of all Googlers who are members of the organization. More granular approvals can be achieved with a CODEOWNERS file where you can specify multiple teams.

Reply:
Yes, either one organization team, if it exists, suffices the needs.

Reply:
Would it make sense to allow for an `org` input where internal approvers are members of the org instead of a team?

Reply:
Yes, organization membership makes sense to me. It can tell the organization already for the repository URL.